schema/migrations/migrate9to10.rb

```diff
@@ -29,5 +31,25 @@ def migrate
         end
       end
     end
+
+    if @hash.key?("packages")
```

Comment:
I would only run this part if patterns are there, otherwise people might get the warning without them.